src/System.Runtime/tests/System/Reflection/TypeInfoTests.cs

```diff
@@ -0,0 +1,223 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.IO;
+using System.Linq;
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public class TestType : Stream
```

Comment:
Any particular reason why TestType derives from an api type?

I'm not sure where Stream stands in the api bringup effort but if we restore new members to that have to be overridden, you'll have to update this class and the various tests that depend on it.